js/repl/src/App.js

```diff
@@ -0,0 +1,3 @@
+import Repl from './repl/Repl.js';
+
+export default Repl;
```

Comment:
What's the purpose of just exporting `Repl`? Could we just use `repl/Repl.js` directly?